src/main/cpp/subsystems/Conveyor.cpp

```diff
@@ -39,22 +39,18 @@ void Conveyor::BallIntakeExiting()
 }
 void Conveyor::SetConveyor()
 {
-    if(CurrentBallCount() == 0 && !IsBallComing() && IsBallExiting())
+    if((IsBallComing() && BallCount() < 5) /* TODO: add in here: || (BallCount() > 0 && IsShooting()) */ )
```

Comment:
Add an `IsShooting()` method to Conveyor that returns `false`.  With the todo in that function to calling the shooter.